bin/verify-exercises

```diff
@@ -20,16 +20,24 @@
 
 # Verify the Concept Exercises
 for concept_exercise_dir in ./exercises/concept/*/; do
-    if [ -d $concept_exercise_dir ]; then
-        echo "Checking $(basename "${concept_exercise_dir}") exercise..."
-        # TODO: run command to verify that the exemplar solution passes the tests
+    if [ -d "$concept_exercise_dir" ]; then
+        slug=$(basename "${practice_exercise_dir}")
+        echo "Checking ${slug} exercise..."
+        mv "${concept_exercise_dir}/${slug}".sql "${concept_exercise_dir}/${slug}"_backup.sql
+        cp "${concept_exercise_dir}"/.meta/exemplar.sql "${concept_exercise_dir}/${slug}".sql
+        bin/verify-exercise-example "${slug}" "${concept_exercise_dir}" "${concept_exercise_dir}"
```

Comment:
Do you want to collect/count/check if exercises fail the validation?

You can use a count.

```bash
failures=0
...
bin/verify-exercise-example "${slug}" "${concept_exercise_dir}" "${concept_exercise_dir}" && (( failures++ ))
...

if (( failures )); then
  echo "${failures} tests failed!"
  exit 1
fi
exit 0

Reply:
Yes, that might be useful.